test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -152,7 +179,16 @@ func describeQatDpdkPlugin() {
 		})
 
 		ginkgo.It("deploys a compress pod (openssl) requesting QAT resources [App:openssl]", func(ctx context.Context) {
-			runCpaSampleCode(ctx, f, compression, resourceName)
+			command := []string{
+				"cpa_sample_code",
+				"runTests=" + strconv.Itoa(compression),
+				"signOfLife=1",
+			}
+			pod := createPod(ctx, f, "qat-engine-testapp", resourceName, "intel/openssl-qat-engine:devel", command)
+
+			ginkgo.By("waiting the qat-engine-testapp pod for the resource" + resourceName.String() + "to finish successfully")
```

Comment:
```suggestion
			ginkgo.By("waiting the qat-engine-testapp pod for the resource " + resourceName.String() + " to finish successfully")
```

Reply:
@hj-johannes-lee there are other places that need the same fix too

Reply:
Thanks, fixed.!